.cicd/platforms/pinned/centos-7.7-pinned.dockerfile

```diff
@@ -10,8 +10,9 @@ RUN yum update -y && \
     python python-devel rh-python36 file libusbx-devel \
     libcurl-devel patch vim-common jq glibc-locale-source glibc-langpack-en \
     libuuid-devel libtasn1-devel expect socat libseccomp-devel iproute && \
-    yum clean all && rm -rf /var/cache/yum
-# install erlang and rabbitmq
+    yum clean all && rm -rf /var/cache/yum && \
+    yum install -y python3
```

Comment:
Why are you invoking this install separate from the list that ends on line 12? This undoes the cache pruning on line 13 also....